plugin/daemon_example/daemon_example.cc

```diff
@@ -77,14 +77,14 @@ pthread_handler_t mysql_heartbeat(void *p)
     daemon_example_plugin_init()
 
   DESCRIPTION
-    Starts up heartbeatbeat thread
+    Starts up heartbeat thread (mysql_heartbeat)
 
   RETURN VALUE
     0                    success
     1                    failure (cannot happen)
 */
 
-static int daemon_example_plugin_init(void *p __attribute__ ((unused)))
+static int daemon_example_plugin_init(void *)
```

Comment:
This should be cleaned up in `daemon_example_plugin_deinit()` as well. The `#define __attribute__` can then be removed.